src/mdast/tokensToMyst.ts

```diff
@@ -379,6 +381,28 @@ const defaultMdast: Record<string, Spec> = {
       };
     },
   },
+  parsed_directive: {
+    type: 'directive',
+    getAttrs(t) {
+      let opts = t.meta?.opts;
+      if (!opts || !Object.keys(opts).length) {
+        opts = undefined;
+      } else {
+        if (opts.class) opts.class = opts.class.join(' ');
+        Object.keys(opts).forEach((k) => {
+          // Handle flags, where option is simply present.
+          // This simple solution is very unlikely to be sufficient long term.
+          if (opts[k] === null) opts[k] = true;
```

Comment:
This handling of `opts` is really messy and should likely be handled elsewhere (`markdown-it-docutils`?). However, it doesn't affect the parsed content, it's only for adding options to the `directive` node, so it doesn't hugely matter...

Reply:
Yeah, there are a few bugs like this in docutils:
https://github.com/executablebooks/markdown-it-docutils/issues/33